src/helpers/userBehaviour.js

```diff
@@ -0,0 +1,18 @@
+const HAS_VISITED_FEEDBACK_PAGE = 'HAS_VISITED_FEEDBACK_PAGE';
+const NAVIGATION_COUNT = 'NAVIGATION_COUNT';
+
+export function hasVisitedFeedbackPage() {
+  return localStorage.getItem(HAS_VISITED_FEEDBACK_PAGE) === true.toString();
+}
+
+export function rememberVisitToFeedbackPage() {
+  localStorage.setItem(HAS_VISITED_FEEDBACK_PAGE, true.toString());
+}
+
+export function navigationCount() {
+  return parseInt(localStorage.getItem(NAVIGATION_COUNT), 10) || 0;
+}
+
+export function incrementNavigationCount() {
+  localStorage.setItem(NAVIGATION_COUNT, navigationCount() + 1);
```

Comment:
Very minor: I like to explicitly say `window.variableName` for any variable that I am expecting to exist on `window`. I don't think it really matters for `localStorage`, hence "minor". But for things like `innerHeight` or `scrollTo` it's weird to use it without `window`, so for consistency, doing `window.localStorage` is nice.